pkg/server/container_execsync.go

```diff
@@ -55,6 +56,22 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 		return nil, fmt.Errorf("container %q is in %s state", id, criContainerStateToString(meta.State()))
 	}
 
+	// Get exec process spec.
```

Comment:
above comment // Get container config from container store .. should be changed to // Get metadata for container from our container store .. or something like that